impeller/renderer/compute_command.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/renderer/compute_command.h"
+
+#include "impeller/base/validation.h"
+#include "impeller/renderer/formats.h"
+#include "impeller/renderer/vertex_descriptor.h"
+
+namespace impeller {
+
+bool ComputeCommand::BindResource(ShaderStage stage,
+                                  const ShaderUniformSlot& slot,
+                                  const ShaderMetadata& metadata,
+                                  BufferView view) {
+  FML_DCHECK(stage == ShaderStage::kCompute);
```

Comment:
Instead of a DCHECK that is all but useless for non-engine developers, just return false?